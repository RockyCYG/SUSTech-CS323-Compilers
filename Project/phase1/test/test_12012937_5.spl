int poor(){
	(mercy = mercy) || (on)*me;
	set&&fire = 2!=history+0;
}
char I(){
	int m = 'breaking my own rules';
	I = ' '+m == crying + as + a +fool;
	Tall * stories >= (on[the] - page);
	-Short.glories_on 5the faint;
	{I / m && close - enough='\n'=to >=touch;}
	{
		int but - I = never || cared + for +love;
		{
			It - 's' < a > church && of - burnt * romances;
			and = I = 'm' /(-2 -4 -got||to -pray(
			
				it [s] = a, solo(song), and + it[s] || only (for, the=brave));{
			}
		}
	}
}
