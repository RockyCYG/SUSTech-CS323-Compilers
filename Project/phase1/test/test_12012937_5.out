Error type A at Line 6: unknown lexeme 'breaking my own rules'
Error type A at Line 9: unknown lexeme 5the
Error type A at Line 10: unknown lexeme '\n'
Error type B at Line 12: syntax error, probably Missing SEMI
Error type B at Line 15: syntax error, probably missing RP
